<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-VU0KHJRO" ignored="1" name="Default test" tests="16" failures="1" timestamp="2020-11-30T23:51:53 IST" time="14.459" errors="0">
  <testcase name="method1" time="0.04" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method3" time="0.004" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method4" time="4.033" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method5" time="0.006" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method5" time="0.007" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method5" time="0.009" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method5" time="0.004" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method5" time="0.003" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method6" time="10.021" classname="tests.TestNGCRUDWithAllTestProperties">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method tests.TestNGCRUDWithAllTestProperties.method6() didn&amp;apos;t finish within the time-out 10000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method tests.TestNGCRUDWithAllTestProperties.method6() didn't finish within the time-out 10000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:371)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:282)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:605)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- method6 -->
  <testcase name="method7" time="0.006" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method8" time="0.006" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method9" time="0.003" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method10" time="0.002" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method11" time="0.003" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method12" time="0.002" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method13" time="0.004" classname="tests.TestNGCRUDWithAllTestProperties"/>
  <testcase name="method2" time="0.0" classname="tests.TestNGCRUDWithAllTestProperties">
    <ignored/>
  </testcase> <!-- method2 -->
</testsuite> <!-- Default test -->
